judgels-frontends/raphael/src/conf.ts

```diff
@@ -19,6 +19,7 @@ export interface AppConfig {
   apiUrls: ApiUrlsConfig;
   tempHome: TempHomeConfig;
   welcomeBanner: WelcomeBannerConfig;
+  termsAndConditions: string;
```

Comment:
I think, it is better to make it like this:

```
export interface TermsAndConditionsConfig {
  contest: string;
}
```

Because we might want to use TnC for other things as well.